mutator/loop/range_break.go

```diff
@@ -0,0 +1,79 @@
+package loop
+
+import (
+	"go/ast"
+	"go/token"
+	"go/types"
+
+	"github.com/avito-tech/go-mutesting/mutator"
+)
+
+func init() {
+	mutator.Register("loop/range_break", MutatorLoopRangeBreak)
+}
+
+func checkIsRangeStatement(node ast.Stmt) bool {
+	switch node.(type) {
+	case *ast.RangeStmt:
+		return true
+	}
+
+	return false
+}
+
+// MutatorLoopRangeBreak implements a mutator to add a break to range-loop body.
+func MutatorLoopRangeBreak(_ *types.Package, _ *types.Info, node ast.Node) []mutator.Mutation {
+	var list []ast.Stmt
+
+	switch n := node.(type) {
+	case *ast.BlockStmt:
+		list = n.List
+	case *ast.CaseClause:
+		list = n.Body
```

Comment:
can u kindly explain - `CaseClause` how we can get a RangeStmt inside?
may be some code examples ?

Reply:
any range-based for inside case block
	
	case "darwin":
		var cow = []float64{1.0, 2.0, 3.0}
		for _, v := range cow {
			fmt.Print(v)
		}